### PR TITLE
Send Exit Survey Results to Empirca

### PR DESCRIPTION
**Pull request by Alan-Qiao**
Saves the survey results, which are in the form of a JSON object, to the player object as a key-value pair where the key is "ExitSurvey", and the value is the object converted to a string.

Manual testing confirms that this key-value pair is saved in tajriba.json as an attribute associated with the player object.

Cypress testing also confirms that the inputted survey response is present in tajriba.json (converted to txt due to invalid json syntax parsing errors)

[Linked Issue](https://github.com/Watts-Lab/surveys/issues/4)

### CODE REVIEW

File: client/cypress/integration/ExampleExitSurvey.test.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        cy.get('textarea[aria-label="What do you miss and what was disappointing in your experience with us?"]').type('Discussion too short');
+        cy.get('input[value="Complete"]').click();
+        // temporary solution with testing existence in tajriba.json
+        cy.exec('cd .. && cd .empirica/local && mv tajriba.json tajriba.txt');
+        // cy.readFile('../.empirica/local/tajriba.txt').then(data => {
+        //     expect(data.includes(JSON.stringify(expectedJSON))).to.be.true;
+        // });
+        cy.readFile('../.empirica/local/tajriba.txt').then(data => {
+            console.log(data);
+            expect(data.includes(expectedString)).to.be.true;
```
Comment (JamesPHoughton) [inline client/cypress/integration/ExampleExitSurvey.test.jsx:50]: Nice. I'm glad we can do this...

**Review by JamesPHoughton — CHANGES_REQUESTED** [review]
Kimberly has done some updates to remove most of the example tests in /integration, might be a good idea to update your PR from either her branch or (I think) main, which have those changes.

**Comment by JamesPHoughton** [comment]
Will close https://github.com/Watts-Lab/surveys/issues/4

**Comment by kepstein23** [comment]
I merged main into the branch and all of the example tests are removed now!

**Comment by JamesPHoughton** [comment]
Looks like cypress tests got added also at the main repo level, instead of just in the /client folder. I'll edit to move to the /client folder just to keep things consistent.

**Review by JamesPHoughton — APPROVED** [review]

**Pull request merged**